### ISSUE TITLE
Internal Data Structures

### ISSUE THREAD

**Issue by Darrah** [beloitcollegecomputerscience/BlockadeGames#3]
Resolve the two different approaches to the internal data structures.

**Comment by Darrah** [comment]
Looking at converting Python to Java.
Need names of classes, and names of methods, and parameters for those methods.

**Comment by Darrah** [comment]
In progress.

**Comment by Jerry2001** [comment]
#15 I add a few initial classes to the game but other classes are pretty dependent on #13

**Comment by Darrah** [comment]
Waiting for approval of the Graph class pull request.

**Comment by Jerry2001** [comment]
Current branch in progress is [playing_piences_fundamental](https://github.com/beloitcollegecomputerscience/BlockadeGames/tree/playing_piences_fundamental)

**Comment by Darrah** [comment]
Code completed. Needs to be merged with the project.

**Comment by Jerry2001** [comment]
Code was merged into the main branch. Three teams are working together to hook the animation, GUI, and data structure together.

**Comment by Jerry2001** [comment]
JUnit for DS code is in this branch [junit-ds](https://github.com/beloitcollegecomputerscience/BlockadeGames/tree/junit-ds)
